README.md

```diff
@@ -67,15 +67,15 @@ Build UI files:
   $ ansible-events-ui
 ```
 
-Visit this url: http://localhost:8080/docs#/auth/register_register_api_auth_register_post
+Visit this url: http://localhost/api/docs/auth/register_register_api_auth_register_post
```

Comment:
Why do you remove the port? this section is for the development env, which it uses that port

Reply:
Now when you run with docker compose you no longer need to use port 8080, it is routed through port 80 by default

Reply:
This has been foxed in the README